packages/contracts-bedrock/src/L2/L2StandardBridge.sol

```diff
@@ -52,20 +52,23 @@ contract L2StandardBridge is StandardBridge, ISemver {
         bytes extraData
     );
 
-    /// @custom:semver 1.7.0
-    string public constant version = "1.7.0";
+    /// @custom:semver 1.8.0
+    string public constant version = "1.8.0";
 
     /// @notice Constructs the L2StandardBridge contract.
     /// @param _otherBridge Address of the L1StandardBridge.
-    constructor(address payable _otherBridge)
-        StandardBridge(payable(Predeploys.L2_CROSS_DOMAIN_MESSENGER), _otherBridge)
-    {
-        initialize();
+    constructor(address payable _otherBridge) StandardBridge() {
```

Comment:
we don't need an arg here, we can hardcode `address(0)` for the singleton

Reply:
this would require changing the abi layout for the l2 contract (which we may not want to affect). should I go ahead and drop the arg here?

Reply:
Yes its fine because technically we set the storage directly into storage without execution